### DISCUSSION TITLE
Add Italics

### DISCUSSION THREAD

**Discussion by joshistoast** [BeardedBear/bearded-theme#105] [Ideas]
italics are love,
italics are life

**Comment by joshistoast** [comment]
I've already read some of the closed PRs and issues and the general consensus is "maybe down the road" and "this method seems inefficient".

If the author feels negatively towards italics, perhaps even add a short guide on configuring the vscode settings to use italics 🤷

**Reply by BeardedBear** [reply]
Hi! I don't have any problem with italics (besides, in the first versions of the theme there were quite a few italicized parts) but I received more requests complaining about italics than requests to leave them.

As for a little tutorial to add them yourself, it's a good idea, I'll think about it.

My idea would be to be able to add options to the theme configuration rather than duplicate the long list of variations already present, but I need to take the time to learn how to do that :)

**Reply by babakfp** [reply]
Hi. Can I remove italics? Comments have italic style.

For people that want both, if you want to support, you can do something like this (no-italic/italic):

![image](https://github.com/user-attachments/assets/49cca09f-ea8a-464b-aba9-55833825d977)
